packages/core/tsconfig.json

```diff
@@ -4,8 +4,7 @@
     "src/"
   ],
   "compilerOptions": {
-    "rootDir": "./src",
```

Comment:
This would make ts-node+tsc complain that the `test/` folder wasn't inside the root. I think the `"include"` array is enough.